tuichain/api/views/loans.py

```diff
@@ -116,6 +116,58 @@ def create_loan_request(request):
     )
 
 
+@api_view(["PUT"])
+@permission_classes((IsAuthenticated,))
+def cancel_loan(request, id):
+    """
+    Cancel a Loan
+
+    Parameters
+    ----------
+    id : integer
+
+        Loan's identifier.
+
+    Returns
+    -------
+    201
+        Loan canceled successfully.
+
+    403
+        Cannot cancel a loan that you don't own
+
+    403
+        Loan cannot be canceled.
+
+    404
+        Loan not found.
+
+    """
+    user = request.user
+    loan = Loan.objects.filter(id=id).first()
+
+    if user != loan.student and not user.is_superuser:
+        return Response(
+            {"error": "Cannot cancel a loan that you don't own"},
+            status=HTTP_403_FORBIDDEN,
+        )
+
+    if loan is None:
+        return Response({"error": "Unexistent Loan"}, status=HTTP_404_NOT_FOUND)
+
+    if loan.state != LoanState.APPROVED.value:
+        return Response(
+            {"error": "The given Loan cannot be canceled"},
+            status=HTTP_403_FORBIDDEN,
+        )
+
+    controller.loans.get_by_identifier(LoanIdentifier(loan.identifier)).cancel()
+
+    return Response(
+        {"message": "Loan has been canceled"}, status=HTTP_201_CREATED
```

Comment:
Is 200 more adequate?

Reply:
I thought the same but we are using 201 on the others like validate and reject
Should I change them all?